tests/ui/test_ui_tools.py

```diff
@@ -613,8 +615,10 @@ def test_keypress_SEARCH_TOPICS(self, mocker, topic_view, key):
     def test_keypress_GO_BACK(self, mocker, topic_view, key):
         size = (200, 20)
         mocker.patch(VIEWS + '.TopicsView.set_focus')
+        mocker.patch.object(topic_view.topic_search_box, 'reset_search_text')
         topic_view.keypress(size, key)
         topic_view.set_focus.assert_called_once_with("body")
+        topic_view.topic_search_box.reset_search_text.called
```

Comment:
Try removing the function calls, and see if this works; that's often why having a failing test first is useful :)

To reiterate, some mock attributes are of the form `some_mock.assert_something`, while others need `assert some_mock.something`. I often refer to the `mock` pages.